### PR TITLE
Add kernel examples

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: figure/plot-cross-section-checkerboard.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Plot kernel (b)
+gmt subplot set 1
+gmt basemap -R0.0/240.0/0.0/36.0 -JX?/-?
+awk ' { print $1,-1.0*$3,$4 }' ../inversion/kernel | gmt surface -R0.0/240.0/0.0/36.0 -I0.2 -Gtomo.grd
+T=$(gmt grdinfo -T+a5+s tomo.grd)
+gmt makecpt -Cpsrgb.cpt $T -I
+gmt grdimage tomo.grd
+gmt colorbar -DJRM+w1.8c/0.2c+e -Ba
+
+# Plot true velocity model (a)
```
Comment (core-man) [inline figure/plot-cross-section-checkerboard.sh]:
```suggestion
# Plot true velocity model (c)
```

File: figure/plot-cross-section-checkerboard.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+# Plot kernel (b)
+gmt subplot set 1
+gmt basemap -R0.0/240.0/0.0/36.0 -JX?/-?
+awk ' { print $1,-1.0*$3,$4 }' ../inversion/kernel | gmt surface -R0.0/240.0/0.0/36.0 -I0.2 -Gtomo.grd
+T=$(gmt grdinfo -T+a5+s tomo.grd)
+gmt makecpt -Cpsrgb.cpt $T -I
+gmt grdimage tomo.grd
+gmt colorbar -DJRM+w1.8c/0.2c+e -Ba
+
```
Comment (core-man) [inline figure/plot-cross-section-checkerboard.sh:44]:
```suggestion
awk '{print $3, $2}' ../data/receivers | gmt plot -Si0.3c -Gblue -N
awk '{print $3, -$2}' ../data/sources | gmt plot -Sa0.2c -Gred -W0.5p,white

```

File: figure/plot-cross-section-checkerboard.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# Plot velocity model (a)
+gmt subplot set 0
+gmt basemap -R0.0/240.0/0.0/36.0 -JX?/-? 
+awk ' { print $1,$2,$5 }' dvelCut001 | gmt surface -R0.0/240.0/0.0/36.0 -I0.2 -Gtomo.grd
+T=$(gmt grdinfo -T+s tomo.grd)
+gmt makecpt -Cpsrgb.cpt $T -I
+gmt grdimage tomo.grd
+gmt colorbar -DJRM+w1.8c/0.2c+e -Ba+u"%"
+
```
Comment (core-man) [inline figure/plot-cross-section-checkerboard.sh:32]:
```suggestion
awk '{print $3, $2}' ../data/receivers | gmt plot -Si0.3c -Gblue -N
awk '{print $3, -$2}' ../data/sources | gmt plot -Sa0.2c -Gred -W0.5p,white

```

File: commandCenter/cleanup.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+cd ../data/
+rm -vf evtstaminmax nevtstaray receivers sources traveltimeReceiverGathers traveltimeReceiverGathers_synthetic
+
+cd ./dataSelection/1_picks/
+rm -vf largeEarthquakes  obstime-rec-gather  obstime-src-gather  phaseDataInResearchRegion
+cd ../../
+
+cd ../figure/
+rm -vf dvelCut* tomo.grd
```
Comment (core-man) [inline commandCenter/cleanup.sh]: What's vz1d? Remove?
```suggestion
rm -vf dvelCut* dveltrueCut001 mesh.mod tomo.grd vz1d
```

File: commandCenter/cleanup.sh [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+cd ../inversion/
+rm -vf adjtfield* kernel residual0* velocity3d0*
```
Comment (core-man) [inline commandCenter/cleanup.sh]:
```suggestion
rm -vf adjtfield* kernel residual0* velocity3d0*

```

File: README.md [hunk 6]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
 ```
 
 The Bash script will call `zCutVelocity.f90` and the gmt script `vcut.gmt`.
+
```
Comment (core-man) [inline README.md:152]:
```suggestion

----

```

File: README.md [hunk 7]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
...
+## Example
```
Comment (core-man) [inline README.md]:
```suggestion
## Examples
```

File: README.md [hunk 8]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
...
+
+### Example1 Calculate and visualize individual kernel
```
Comment (core-man) [inline README.md]:
```suggestion
### Example1: Calculate and visualize individual kernel

```

File: README.md [hunk 9]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
...
+$ ./workflow_inversion.sh
+```
+Then we can plot the result, select model index 1 and velocity perturbation bound 0.06.
+```
+$ cd ../figure/
+$ ./plot-cross-section.sh
+```
+In the image, (a) is velocity model after 1 round of iteration, (b) is the individual kernel, (c) is true velocity model. Red stars denote sources, blue triangles denote receivers.
+
+### Example2 Calculate and visualize event kernel
```
Comment (core-man) [inline README.md]:
```suggestion
### Example2: Calculate and visualize event kernel

```

File: README.md [hunk 10]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
...
+$ ./workflow_obstime.sh
+$ ./workflow_inversion.sh
+```
+Then we can plot the result,select model index 1 and velocity perturbation bound 0.06.
+```
+$ cd ../figure/
+$ ./plot-cross-section.sh
+```
+In the image, (a) is velocity model after 1 round of iteration, (b) is the event kernel, (c) is true velocity model. Red stars denote sources, blue triangles denote receivers.
+### Example3 Calculate and visualize misfit kernel
```
Comment (core-man) [inline README.md]:
```suggestion

### Example3: Calculate and visualize misfit kernel

```

File: README.md [hunk 11]
```diff
@@ -32,7 +32,9 @@ $ brew install gfortran
 [Generic Mapping Tools (GMT)](https://www.generic-mapping-tools.org/) is used to plot figures.
 Refer to [[GMT reference book in Chinese](https://docs.gmt-china.org/latest/install/)] for GMT installation and usage.
 
-## Part I: Data
+## Program document
+
+### Part I: Data
 
 ```
 $ cd data
```
Comment (core-man) [inline README.md]:
```suggestion
$ cd ./data/
```

File: README.md [hunk 12]
```diff
@@ -77,7 +79,7 @@ $ ./xpara
 $ rm xpara
 ```
 
-## Part III: Model
+### Part III: Model
 
 ```
 cd ../model
```
Comment (core-man) [inline README.md]:
```suggestion
cd ../model/
```

File: README.md [hunk 13]
```diff
@@ -98,7 +100,7 @@ $ ./xvel
 $ rm xvel
 ```
 
-## Part IV: Mesh
+### Part IV: Mesh
 
 ```
 cd ../mesh
```
Comment (core-man) [inline README.md]:
```suggestion
cd ../mesh/
```

File: README.md [hunk 14]
```diff
@@ -122,7 +124,7 @@ $ ./xmesh
 $ rm xmesh regmesh.mod
 ```
 
-## Part V: CommandCenter
+### Part V: CommandCenter
 
 ```
 cd ../commandCenter
```
Comment (core-man) [inline README.md]:
```suggestion
cd ../commandCenter/
```

File: commandCenter/cleanup.sh [hunk 15]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```
Comment (core-man) [inline commandCenter/cleanup.sh:1]: Shall we remove those files in `model`:

- velocity2d_true
- velocity3d
- velocity3d_true

File: README.md [hunk 16]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
...
+
+### Example2 Calculate and visualize event kernel
+First, clean the output of previous example.
+```
+$ cd ../commandCenter/
+$ ./cleanup.sh
+```
+Then run the following code.
+```
+$ cd ./data/
```
Comment (core-man) [inline README.md]:
```suggestion
$ cd ../data/
```

File: README.md [hunk 17]
```diff
@@ -144,3 +146,62 @@ $ ./plot-cross-section.sh
...
+In the image, (a) is velocity model after 1 round of iteration, (b) is the event kernel, (c) is true velocity model. Red stars denote sources, blue triangles denote receivers.
+### Example3 Calculate and visualize misfit kernel
+First, clean the output of previous example.
+```
+$ cd ../commandCenter/
+$ ./cleanup.sh
+```
+Then run the following code.
+```
+$ cd ./data/
```
Comment (core-man) [inline README.md]:
```suggestion
$ cd ../data/
```

**Review by HouseJaay — APPROVED** [review]

**Pull request merged**